crates/signer-ledger/src/signer.rs

```diff
@@ -376,7 +392,7 @@ mod tests {
         test_sign_tx_generic(&mut tx).await;
     }
 
-    async fn test_sign_tx_generic(tx: &mut SignableTx) {
+    async fn test_sign_tx_generic(tx: &mut dyn SignableTransaction<Signature>) {
```

Comment:
do we have alias that doesnt require us to specify the generic?

Reply:
u cant forward declare them on traits

Reply:
`type SignableTx = dyn SignableTransaction<Signature>`
but it's a test